src/haxelib/client/Main.hx

```diff
@@ -611,8 +616,12 @@ class Main {
 		return password;
 	}
 
+	function getWritableRepository(reps: Array<String>):String {
+		return reps[0];
```

Comment:
When `haxelib setup` is never run yet, e.g. when haxe is just installed, this may point to the system repo (`HAXELIB_PATH`)?